lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,12 @@
+module ActionController
```

Comment:
Yeah, let's put it in `active_model_serializers/serialization_context` and I'm OK to merge.